src/components/Inputs/Input.tsx

```diff
@@ -0,0 +1,28 @@
+import { cn } from '@/libs/cn';
+
+export default function Input({
+  ref,
+  className,
+  type,
+  placeholder = '',
+  name,
+  id,
+  ...props
+}: React.ComponentProps<'input'>) {
+  return (
+    <div>
+      <input
+        ref={ref}
+        className={cn(
+          'text-md placeholder:text-md w-1/2 rounded-2xl border border-gray-300 bg-white px-5 py-3 placeholder:text-gray-500 focus:outline-none',
+          className,
+        )}

```

Comment:
외부에서 `className`을 계산하면 가독성이 더 좋아질 것 같습니다!


```suggestion
        const className = cn(
          'text-md placeholder:text-md w-1/2 rounded-2xl border border-gray-300 bg-white px-5 py-3 placeholder:text-gray-500 focus:outline-none',
          className,
        );
        
        // 사용할 때
        className={className}
```